blamo/lib/LogInfo/loginfo.dart

```diff
@@ -50,101 +71,206 @@ class _LogInfoPageState extends State<LogInfoPage> {
                 child: Column(
                 children: <Widget>[ 
                   FormBuilderTextField(
+                    textInputAction: TextInputAction.next,
                     attribute: 'title',
                     validators: [],
                     decoration: InputDecoration(labelText: "Project Name"),
-                  ),
-                  FormBuilderDateTimePicker(
-                    attribute: "start-date",
-                    inputType: InputType.date,
-                    validators: [],
-                    format: DateFormat("dd-MM-yyyy"),
-                    decoration: InputDecoration(labelText: "Start Date"),
-                  ),
-                  FormBuilderDateTimePicker(
-                    attribute: 'end-date',
-                    inputType: InputType.date,
-                    validators: [],
-                    format: DateFormat('dd-MM-yyyy'),
-                    decoration: InputDecoration(labelText: "End Date"),
+                    onFieldSubmitted: (v){
+                      FocusScope.of(context).requestFocus(formNodes[1]);
+                    },
                   ),
                   FormBuilderTextField(
-                    attribute: 'driller',
+                    textInputAction: TextInputAction.next,
+                    focusNode: formNodes[1],
+                    attribute: 'purpose',
                     validators: [],
-                    decoration: InputDecoration(labelText: "Driller"),
+                    decoration: InputDecoration(labelText: "Purpose"),
```

Comment:
The updated loginfo fields don't line up with the output PDF values.

loginfo.dart
Project Name
Purpose
Highway
County
Northing
Easting
Equipment
Driller
Project Geologist
Recorder
Start Date
End Date
Total Depth
Ground Elevation
Tube Height
Hole No.
E.A. No
Key No.
Start Card No.
Bridge No.

PDF LogInfo class
 objectID
 testType
 project
 number
 client
 lat
 long
 location
 elevationDatum
 boreholeID
 startDate
 endDate
 surfaceElevation
 contractor
 method
 loggedBy
 checkedBy